api/package.json

```diff
@@ -22,7 +22,7 @@
     "db:resetup": "psql -c 'DROP DATABASE IF EXISTS bloom_prisma WITH (FORCE);' && psql -c 'CREATE DATABASE bloom_prisma;' && psql -d bloom_prisma -c 'CREATE EXTENSION IF NOT EXISTS \"uuid-ossp\";'",
     "db:migration:run": "yarn prisma migrate deploy",
     "db:seed:production": "npx prisma db seed -- --environment production",
-    "db:seed:staging": "npx prisma db seed -- --environment staging",
+    "db:seed:staging": "npx prisma db seed -- --environment staging --jurisdictionName Bloomington",
```

Comment:
thought: The original reason we didn't add the `jurisdictionName` flag here is because we anticipated using these seeding scripts in our new environments to seed initial data for a new jurisdiction. So we would call this with the new jurisdictionName. Especially with the `production` version of the seeds command

However, I didn't realize that we could still do that because it just takes the last instance of it so 

`yarn db:seed:staging --jurisdictionName newName`

turns into

`npx prisma db seed -- --environment staging --jurisdictionName Bloomington --jurisdictionName newName`

and "newName" is the one chosen. Pretty neat!